project-koku/koku
# adjust required AWS columns

## Jira Ticket

[COST-3991](https://issues.redhat.com/browse/COST-3991)

## Description

This change will remove "productinstancetype", "productvcpu", "productmemory" from required columns and add them as optionl. QE have at least one account that does not include these in their real report.

## Testing

1. Checkout Branch
2. Restart Koku
3. Create an AWS source
4. generate some data without these optional columns included
5. see everything ingests correctly.

Test 2
Also check the reverse works, E.I those optional columns do in fact get collected and processed correctly.
 
## Notes
This also addresses a minor bug sending files to S3 without `.csv` extensions
...
